app/src/main/java/in/antaragni/ant/MainActivity.java

```diff
@@ -25,17 +39,19 @@
 import in.antaragni.ant.fragments.MapFragment;
 import in.antaragni.ant.fragments.ScheduleFragment;
 
-public class MainActivity extends AppCompatActivity
+public class MainActivity extends AppCompatActivity implements GoogleApiClient.OnConnectionFailedListener
 {
   protected static int HOME = 1;
   protected static int SCHEDULE = 2;
   protected static int EVENTS = 3;
   protected static int MAP = 4;
   protected static int CONTACT = 5;
   protected static int ABOUT = 6;
+  protected static int SIGNOUT=7;
 
   //save our header or result
   private Drawer result = null;
+  private GoogleApiClient CLIENT;
```

Comment:
@saransh7 @santwanaverma15 @Piyush-ag plz note. ALLCAPS are used for constants. plz rename this one.